nvtabular/ops/categorify.py

```diff
@@ -123,6 +124,8 @@ def __init__(
         encode_type="joint",
         name_sep="_",
         search_sorted=False,
+        num_buckets=None,
+        hash_type=None,
```

Comment:
Whats the hash_type argument for?  Can you also update the docstring for the num_buckets argument?

Reply:
@benfred will remove hash_type. I added it for a reason but it was not required afterwards.